testConnectivity.py

```diff
@@ -745,17 +768,22 @@ def testConnectivity(args):
                         name="dacScanTree",
                         nameX="dummy", # temporary name, will be over-ridden
                         nameY=("ADC1" if args.extRefADC else "ADC0"),
-                        dacSelect=-1, #temporary value, will be over-ridden 
+                        dacSelect=-1, #temporary value, will be over-ridden
                         description="GEM DAC Calibration of VFAT3 DAC"
                 )
- 
+
+        printYellow("CAlTree retrieved")
         # Place All Chips Into Run Mode and write correct Iref
         from math import isnan
+        printYellow("isnan imported")
```

Comment:
Seems like this is a debugging statements that are added but no longer needed....?